tests/automation/message_checks_groups.spec.ts

```diff
@@ -18,302 +14,250 @@ import {
+    );
+    await sleepFor(100);
+    await typeIntoInput(aliceWindow1, 'message-input-text-area', testMessage);
 
-  await clickOnElement({
-    window: windowA,
-    strategy: 'data-testid',
-    selector: 'send-message-button',
-  });
-  await sleepFor(1000);
-  await replyTo(windowB, testMessage, testReply);
-  await waitForTextMessage(windowA, testReply);
-});
+    await clickOnElement({
+      window: aliceWindow1,
+      strategy: 'data-testid',
+      selector: 'send-message-button',
+    });
+    await sleepFor(1000);
+    await replyTo({
+      senderWindow: bobWindow1,
+      textMessage: testMessage,
+      replyText: testReply,
+      receiverWindow: aliceWindow1,
+    });
+  },
+);
 
-test('Send long text to group', async () => {
-  const [windowA, windowB, windowC] = await openApp(3);
-  const [userA, userB, userC] = await Promise.all([
-    newUser(windowA, 'Alice'),
-    newUser(windowB, 'Bob'),
-    newUser(windowC, 'Charlie'),
-  ]);
-  const group = await createGroup(
-    'Message checks',
-    userA,
-    windowA,
-    userB,
-    windowB,
-    userC,
-    windowC,
-  );
-  const longText =
-    // eslint-disable-next-line max-len
-    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum quis lacinia mi. Praesent fermentum vehicula rhoncus. Aliquam ac purus lobortis, convallis nisi quis, pulvinar elit. Nam commodo eros in molestie lobortis. Donec at mattis est. In tempor ex nec velit mattis, vitae feugiat augue maximus. Nullam risus libero, bibendum et enim et, viverra viverra est. Suspendisse potenti. Sed ut nibh in sem rhoncus suscipit. Etiam tristique leo sit amet ullamcorper dictum. Suspendisse sollicitudin, lectus et suscipit eleifend, libero dui ultricies neque, non elementum nulla orci bibendum lorem. Suspendisse potenti. Aenean a tellus imperdiet, iaculis metus quis, pretium diam. Nunc varius vitae enim vestibulum interdum. In hac habitasse platea dictumst. Donec auctor sem quis eleifend fermentum. Vestibulum neque nulla, maximus non arcu gravida, condimentum euismod turpis. Cras ac mattis orci. Quisque ac enim pharetra felis sodales eleifend. Aliquam erat volutpat. Donec sit amet mollis nibh, eget feugiat ipsum. Integer vestibulum purus ac suscipit egestas. Duis vitae aliquet ligula.';
-  const testReply = `${userB.userName} replying to long text message from ${userA.userName} in ${group.userName}`;
-  await typeIntoInput(windowA, 'message-input-text-area', longText);
-  await sleepFor(100);
-  await clickOnElement({
-    window: windowA,
-    strategy: 'data-testid',
-    selector: 'send-message-button',
-  });
-  await sleepFor(1000);
-  await replyTo(windowB, longText, testReply);
-  await waitForTextMessage(windowC, longText);
-  await waitForTextMessage(windowC, testReply);
-});
+test_group_Alice_1W_Bob_1W_Charlie_1W(
+  'Send long text to group',
+  async ({
+    alice,
+    bob,
+    aliceWindow1,
+    bobWindow1,
+    charlieWindow1,
+    groupCreated,
+  }) => {
+    const longText =
```

Comment:
Given long text is duplicated I think having it stored once somewhere is a good idea

Reply:
keeping this comment for @burtonemily